worker/process_batch_worker.go

```diff
@@ -191,20 +191,25 @@ func (batchWorker *ProcessBatchWorker) updateJobBatchesInfo(jobID uuid.UUID) err
 	if err != nil {
 		return err
 	}
+	if job.TotalBatches != 0 && job.CompletedBatches == 1 && job.CompletedAt == 0 {
+		job.TagRunning(batchWorker.MarathonDB, "process_batche_worker", "starting")
+	}
 	if job.TotalBatches != 0 && job.CompletedBatches >= job.TotalBatches && job.CompletedAt == 0 {
 		l := batchWorker.Logger.With(
 			zap.String("source", "processBatchWorker"),
 			zap.String("operation", "updateJobBatchesInfo"),
 			zap.Int("totalBatches", job.TotalBatches),
 			zap.Int("completedBatches", job.CompletedBatches),
 		)
+
 		log.I(l, "Finished all batches")
+		job.TagSuccess(batchWorker.MarathonDB, "process_batche_worker", "Finished all batches")
 		job.CompletedAt = time.Now().UnixNano()
 		_, err = batchWorker.MarathonDB.DB.Model(&job).Column("completed_at").Update()
 		if err != nil {
 			return err
 		}
-		at := time.Now().Add(10 * time.Minute).UnixNano()
+		at := time.Now().Add(5 * time.Second).UnixNano()
```

Comment:
You shouldn't change this! I think you did this to have faster feedback while testing locally, to solve this we should parametrize the delay by environment.